namespaces/live.cloud-platform.service.justice.gov.uk/help-with-prison-visits-prod/resources/variables.tf

```diff
@@ -16,6 +16,11 @@ variable "team_name" {
   default     = "hmpps-help-with-prison-visits-live"
 }
 
+variable "environment" {
```

Comment:
Create correct name without breaking anything else that uses the old environment-name variable.